test/lif-test.jl

```diff
@@ -0,0 +1,50 @@
+@testset "lif-test.jl" begin
```

Comment:
No need to create a new `@testset`. See the comment about multiple plot commands. All of this code will produce a single plot, so it can be wrapped in a single `@plottest` command (similar to the excite test). The plot is built iteratively using multiple plot commands, but in the end there will only be a single plot.